code/mob/living/critter/small_animal.dm

```diff
@@ -2584,6 +2584,15 @@ var/list/mob_bird_species = list("smallowl" = /mob/living/critter/small_animal/b
 				return 2
 		return ..()
 
+	specific_emotes(var/act, var/param = null, var/voluntary = 0)
```

Comment:
This should instead be in the already defined `specific_emotes` proc instead of creating a new override.

Reply:
Actually I see there are two lol, I must have done a dumb at some point. You could merge all three into one `specific_emotes` override.

Reply:
merged all three into one `specific_emotes` override